selection/META.yml

```diff
@@ -0,0 +1,9 @@
+links:
+- product: safari
+  url: https://bugs.webkit.org/show_bug.cgi?id=145212
+  results:
+  - test: "*"
```

Comment:
Is every (or at least the overwhelming majority) of tests in `selection/` failing because of this bug?

Reply:
A big majority of those tests fail due to this bug. Check comment https://bugs.webkit.org/show_bug.cgi?id=145212#c9

Reply:
Ah, thank you, great. Sorry I missed that.